pkg/storaged/fsys-tab.jsx

```diff
@@ -548,6 +569,10 @@ export class FilesystemTab extends React.Component {
             }
             if (opt_ro)
                 opts.push("ro");
+            if (opt_nofail)
+                opts.push("nofail");
+            if (opt_netdev)
+                opts.push("netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221108-140731-41f536f4-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)